portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,15 +34,27 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
-  private List<String> comments;
+//   private List<String> comments = new ArrayList<>();
 
-  @Override
-  public void init() {
-    comments = new ArrayList<>();
-  }
+//   @Override
+//   public void init() {
+//     comments = new ArrayList<>();
+//     }
```

Comment:
Don't check in commented out code, just delete it.  You can always retrieve it from the git history